server/middleware/validation/imprisonmentStatusesValidation.test.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response } from 'express'
+import validation from './imprisonmentStatusesValidation'
+
+describe('Imprisonment status validation middleware', () => {
+  let req = { body: {}, flash: jest.fn() } as unknown as Request
```

Comment:
I think @louisenorris is adding mockResponse and mockRequests from BVL for some of the work she has been doing.


It would be better to follow the pattern we use in BVL where we separate out the validation logic from the middleware - by leveraging SRP it means that you only test the interaction with the web aspects of it once -and then each individual validation test becomes **very** simple